nerfstudio/models/nerfacto.py

```diff
@@ -292,6 +301,8 @@ def get_outputs(self, ray_bundle: RayBundle):
             "accumulation": accumulation,
             "depth": depth,
         }
+        if self.config.use_alpha_transparency_training:
```

Comment:
I would suggest making this default and removing the flag from the config. The transparency training will only be used if background_color is "random"

Reply:
The current implementation of the `random` setting picks a random color per ray (not matched to the target). 

Because the target pixel does match the random color, the only way to minimize the loss during training is to have an accumulation of 1 along every training ray. Since it is impossible for the radiance field to match the constantly changing randomly chosen color.

I don't know if this has any advantage over the default setting 'last sample', which picks the color generated by the radiance field the furthest along the ray as background color (AFAIK). Maybe the current `random` setting helps in pushing the radiance field to have a more pronounced background model for unbounded scenes. But some small scale testing I did does not confirm that `random` improved results in any way.

Reply:
That makes sense. Thanks!

Reply:
Do you guys think it is acceptable to replace that feature with alpha transparent training if an alpha channel is present? In that case, we could get rid of the `use_alpha_transparency_training` argument and make it the default. It might be annoying for people that have images with alpha channels but want to train with the original feature.

Reply:
I would remove the use_lapha_transparency_training and make it default. If `background_color=random` is used the target blended background colour should indeed match the randomly generated ray colour. Please correct me if wrong, but this will be compatible with previous behaviour when no transparency in images, but when images are transparent this will achieve the intended behaviour.

Reply:
All correct :) I will change it